package.json

```diff
@@ -12,6 +12,15 @@
         "postinstall": "cd functions && yarn install",
         "test": "jest ."
     },
+    "lint-staged": {
+        "*.{json,md}": [
+            "prettier --write"
+        ],
+        "*.{js,jsx}": [
+            "prettier --write",
+            "eslint"
```

Comment:
Great idea, but you don’t need to add `eslint .` (period)

Reply:
Thanks :). @rluvaton are you suggesting to add it or to leave it as it is? This doesn't include the `.`

Reply:
Do some test commit with eslint error and check if this working

Reply:
Yep seems to be working:
<img width="547" alt="Screen Shot 2020-03-24 at 2 25 33 PM" src="https://user-images.githubusercontent.com/11335191/77468291-5932fa80-6ddb-11ea-8e44-05fa9df083ea.png">